### PR TITLE
docs: Editor package API

### PR DESCRIPTION
**Pull request by shn-srl**

### CODE REVIEW

**Comment by hejtful** [comment]
Once it's finalised, we could just move the whole thing to the README, as the API is the most important piece of info one might be looking for.

File: packages/editor/README.md [hunk 1]
```diff
@@ -2,6 +2,54 @@
...
+
+#### 2. `SerloRenderer`, `type SerloRendererProps`
+
+- **Why Exported/How Used**: `SerloRenderer` is a component provided by the `@serlo/editor` for rendering content in a non-editable format. This is particularly useful for displaying the content to users who are not currently editing or are not allowed to edit.
+- **Long-Term Support**: Will stay
+- **Needs Change?**: Same changes as `SerloEditor`
+
+#### 3. `type BaseEditor`
+
+- **Why Exported/How Used**: This type describes the `editor` render prop provided by the `SerloEditor` component.
```
Comment (LarsTheGlidingSquirrel) [inline packages/editor/README.md:21]: Maybe a code snippet could help here

File: packages/editor/README.md [hunk 2]
```diff
@@ -2,6 +2,54 @@
...
+
+  - `element` - a React node for rendering the editor
+  - `i18n` - for customizing translation strings
+  - `history` - for persisting, undo, redo
+  - `selectRootDocument` - a function for selecting the current state
+
+- **`pluginsConfig` (optional)**: Serlo Editor plugins can be configured to an extent, this configuration is currently done via the `pluginsConfig` prop of the `SerloEditor` component. Each plugin can be configured separately. There are currently two special rules that apply to the Editor in general:
+
+  - `enableTextAreaExercise`: A flag that enables the TextAreaExercise plugin. TextAreaExercise plugin is currently not yet ready for serlo.org, but it is enabled in Edusharing integration. **To be deprecated once more features are added to the TextAreaExercise plugin and it's ready for serlo.org.**
+  - `exerciseVisibleInSuggestion`: A flag that defines if Exercise plugin is visible in Text plugin suggestions. **Not necessary for Serlo Editor package, instead used by serlo.org, could be removed.**
```
Comment (LarsTheGlidingSquirrel) [inline packages/editor/README.md:43]: Maybe add `To be deprecated once enabled on serlo.org.`

**Review by LarsTheGlidingSquirrel — APPROVED** [review]
I like the comments about things soon to be deprecated. Added some minor comments.

**Pull request merged**